android/build.gradle

```diff
@@ -20,6 +21,7 @@ buildscript {
         classpath("com.google.firebase:firebase-crashlytics-gradle:2.3.0")
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
```

Comment:
The changes to this file are not require, they enable the `next` AsyncStorage version 
https://react-native-async-storage.github.io/async-storage/docs/advanced/next